src/components/forms/PlanForm/tests/index.test.tsx

```diff
@@ -1591,13 +1591,13 @@ describe('containers/forms/PlanForm - Dynamic Form Activities', () => {
     const conditions = wrapper.find('.conditions-fieldset');
     expect(conditions.length).toEqual(6);
     expect(conditions.at(0).text()).toMatchInlineSnapshot(
-      `"ConditionsExpression$this.is(FHIR.QuestionnaireResponse) or (($this.type.where(id='locationType').exists().not() or $this.type.where(id='locationType').text = 'Residential Structure') and $this.contained.exists().not())DescriptionStructure is residential or type does not existExpression$this.is(FHIR.Location) or (questionnaire = 'Register_Structure' and $this.item.where(linkId='structureType').answer.value ='Residential Structure')DescriptionApply to residential structures in Register_Structure questionnaires"`
+      `"ConditionsExpression$this.is(FHIR.QuestionnaireResponse) or (($this.type.where(id='locationType').exists().not() or $this.type.where(id='locationType').text = 'Residential Structure') and $this.contained.exists().not())DescriptionStructure is residential or type does not existFamilyExpression$this.is(FHIR.Location) or (questionnaire = 'Register_Structure' and $this.item.where(linkId='structureType').answer.value ='Residential Structure')DescriptionApply to residential structures in Register_Structure questionnaires"`
```

Comment:
why is this changing?

Reply:
This are changing because of the newly added field on the conditions form